### PR TITLE
Add services_to_ips.pxl

### PR DESCRIPTION
**Pull request by yzhao1012**
Returns a list of services that have active traffic to a list remote IPs.

### CODE REVIEW

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Finds services that have traffic to and from a list of IP addresses.
+# Calculates bandwidth in bytes per second (BPS).
+
```
Comment (zasgar) [inline pxl_scripts/px/services_to_ips/services_to_ips.pxl:3]: please add the license header.

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import px
+
+window_size_seconds = 10
+
+df = px.DataFrame('conn_stats')
+
+# Add IPs of the remote endpoint to filter.
+expr = df.remote_addr == '10.4.1.1'
+expr = expr or df.remote_addr == '10.4.1.2'
```
Comment (zasgar) [inline pxl_scripts/px/services_to_ips/services_to_ips.pxl:12]: Can we make this and argument to the script?

**Review by zasgar — CHANGES_REQUESTED** [review]

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl (hunk 2 above)
Comment (philkuz) [reply pxl_scripts/px/services_to_ips/services_to_ips.pxl:12]: We do not support lists as args at the moment, will want to hardcode for now.

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Finds services that have traffic to and from a list of IP addresses.
```
Comment (philkuz) [inline pxl_scripts/px/services_to_ips/services_to_ips.pxl:1]: Also want to add a manifest.yaml. You can copy it from another script and edit it to describe this one.

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl [hunk 4]
```diff
@@ -23,20 +22,20 @@ df = df[expr]
...
+
+df = df.groupby(['pod', 'upid', 'cmd', 'remote_addr']).agg(
+    bytes_sent=('bytes_sent', px.max),
+    bytes_recv=('bytes_recv', px.max),
+)
 
-# Drop intermediate data columns.
-df = df.drop(['bytes_sent_max', 'bytes_sent_min', 'bytes_recv_max', 'bytes_recv_min'])
+# Insert the remote hostname.
+# This should be done after filtering to minimize overhead.
```
Comment (philkuz) [inline pxl_scripts/px/services_to_ips/services_to_ips.pxl:38]: s/This/nslookup/g

**Review by philkuz — APPROVED** [review]

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+'''
+
+import px
+
+df = px.DataFrame('conn_stats')
+
+# Add IPs of the remote endpoint to filter.
+expr = df.remote_addr == '10.4.1.1'
+expr = expr or df.remote_addr == '10.4.1.2'
```
Comment (zasgar) [inline pxl_scripts/px/services_to_ips/services_to_ips.pxl:17]: Where are these IPs from?

Comment (yzhao1012) [reply pxl_scripts/px/services_to_ips/services_to_ips.pxl]: These are place holders.

The values are from my test cluster.

I've revised the format to be consistent with the purpose of asking users to adapt the expression.
Please take a look at the newest commit.

File: pxl_scripts/px/services_to_ips/services_to_ips.pxl (hunk 4 above)
Comment (yzhao1012) [reply pxl_scripts/px/services_to_ips/services_to_ips.pxl]: Done.

**Comment by philkuz** [comment]
I recommend we just use #111 because it has a vis spec + new query language features

**Comment by yzhao1012** [comment]
This is supersedes by https://github.com/pixie-labs/pixie/pull/111

**Pull request closed without merging**